e2e/@types/synpress.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module "@synthetixio/synpress/helpers" {
+  export function prepareMetamask(version: string): Promise<string>;
+}
+
+declare module "@synthetixio/synpress/commands/metamask" {
+  export function initialSetup(
+    browser: unknown,
```

Comment:
Is it possible to get rid of the `unkown` type?  we can benefit from the `import type { BrowserType } from "@playwright/test";`